megamek/src/megamek/common/Mek.java

```diff
@@ -1113,6 +1113,16 @@ public int getJumpMP(MPCalculationSetting mpCalculationSetting) {
         return Math.max(mp, 0);
     }
 
+    public boolean hasChainDrape() {
+        for (MiscMounted mount : getMisc()) {
+            if (!mount.isDestroyed() && mount.getType().hasFlag(MiscType.F_CHAIN_DRAPE)) {
```

Comment:
Reversing these conditional checks would be marginally faster for units with multiple MiscMounted items, although I'm guessing the difference would be nearly immeasurable.  Just something to consider when checking multiple conditionals: the one that is `true` the least should ideally be first, to maximize short-circuit benefits.